src/link/Coff.zig

```diff
@@ -3602,104 +3617,53 @@ fn freeRelocations(coff: *Coff, atom_index: Atom.Index) void {
-        };
-        itab.entries.items[index] = target;
-        try itab.lookup.putNoClobber(allocator, target, index);
-        return index;
+    pub fn getSymbol(import: Import, coff: *const Coff) *const coff_util.Symbol {
+        return coff.getSymbol(import.symbol);
     }
 
-    const Context = struct {
-        coff: *const Coff,
-        /// Index of this ImportTable in a global list of all tables.
-        /// This is required in order to calculate the base vaddr of this ImportTable.
-        index: usize,
-        /// Offset into the string interning table of the DLL this ImportTable corresponds to.
-        name_off: u32,
-    };
-
-    fn getBaseAddress(ctx: Context) u32 {
-        const header = ctx.coff.sections.items(.header)[ctx.coff.idata_section_index.?];
-        var addr = header.virtual_address;
-        for (ctx.coff.import_tables.values(), 0..) |other_itab, i| {
-            if (ctx.index == i) break;
-            addr += @as(u32, @intCast(other_itab.entries.items.len * @sizeOf(u64))) + 8;
-        }
-        return addr;
+    pub fn getSymbolPtr(import: Import, coff: *Coff) *coff_util.Symbol {
+        return coff.getSymbolPtr(import.symbol);
     }
 
-    fn getImportAddress(itab: *const ImportTable, target: SymbolWithLoc, ctx: Context) ?u32 {
-        const index = itab.lookup.get(target) orelse return null;
-        const base_vaddr = getBaseAddress(ctx);
-        return base_vaddr + index * @sizeOf(u64);
+    pub fn getAddress(import: Import, coff: *const Coff) u32 {
+        const header = coff.sections.items(.header)[coff.idata_section_index.?];
+        return header.virtual_address + import.rva;
     }
 
-    const FormatContext = struct {
-        itab: ImportTable,
-        ctx: Context,
-    };
+    const Index = u32;
+};
 
-    fn format(itab: ImportTable, comptime unused_format_string: []const u8, options: std.fmt.FormatOptions, writer: anytype) !void {
-        _ = itab;
-        _ = unused_format_string;
-        _ = options;
-        _ = writer;
-        @compileError("do not format ImportTable directly; use itab.fmtDebug()");
-    }
-
-    fn format2(
-        fmt_ctx: FormatContext,
-        comptime unused_format_string: []const u8,
-        options: fmt.FormatOptions,
-        writer: anytype,
-    ) @TypeOf(writer).Error!void {
-        _ = options;
-        comptime assert(unused_format_string.len == 0);
-        const lib_name = fmt_ctx.ctx.coff.temp_strtab.getAssumeExists(fmt_ctx.ctx.name_off);
-        const base_vaddr = getBaseAddress(fmt_ctx.ctx);
-        try writer.print("IAT({s}.dll) @{x}:", .{ lib_name, base_vaddr });
-        for (fmt_ctx.itab.entries.items, 0..) |entry, i| {
-            try writer.print("\n  {d}@{?x} => {s}", .{
-                i,
-                fmt_ctx.itab.getImportAddress(entry, fmt_ctx.ctx),
-                fmt_ctx.ctx.coff.getSymbolName(entry),
-            });
+fn addImport(coff: *Coff, allocator: Allocator, target: SymbolWithLoc) !Import.Index {
```

Comment:
```suggestion
fn addImport(coff: *Coff, gpa: Allocator, target: SymbolWithLoc) !Import.Index {
```

Let's use this convention, please - it will really help when refactoring code.